docs/src/makie.md

```diff
@@ -0,0 +1,66 @@
+# Makie.jl plots
+
+Compared to Gadfly.jl and StatsPlots.jl, Makie.jl is the most flexible plotting library that you can use.
+On this page is an example function for plotting with Makie.jl which you can use directly or further tweak to your needs.
+
+```@example makie
+using CairoMakie
+using DataFrames
+using MCMCChains
+
+chns = Chains(randn(300, 5, 3), [:A, :B, :C, :D, :E])
+```
+
+```@example makie
+function plot_chains(chns; density_func=density!)
+    params = names(chns, :parameters)
+
+    df = DataFrame(chns)
```

Comment:
I guess this can be avoided. I'm more familiar with the DataFrames data manipulations than directly handling MCMCChains, so that is why I do this and think that it's reasonable for this example